### PR TITLE
Se classification

### PR DESCRIPTION
**Pull request by feigaodm**
This is a PR to re-classify single electron S2s. Detailed note is [here](https://xe1t-wiki.lngs.infn.it/doku.php?id=xenon:xenon1t:analysis:subgroup:cuts:meetings:20180126:se_classification).

Here is a key plot  from the note. 
screen shot 2018-01-28 at 3 30 04 pm

In addition, I'm adding a training/validation accuracy plot here. The actual training size is ~150k events.
screen shot 2018-01-28 at 4 47 12 pm

### CODE REVIEW

**Review by pdeperio — APPROVED** [review]
Thanks, I also tested and doesn't crash

**Comment by feigaodm** [comment]
@tunnell Do you know why there is still lint issue although I added (# noqa) at the end of each sentence.

File: lax/lichens/sciencerun0.py [hunk 1]
```diff
@@ -873,3 +875,35 @@ def _process(self, df):
                                      (13.719 * np.exp(-df['s2'] / 55.511) + 3.014)) &
                                     (df['s2'] <= 300))))
         return df
+
+
+class SingleElectronS2s(Lichen):  # noqa
+    """To classify S1s from single electron S2s.
+    Information: xenon:xenon1t:analysis:subgroup:cuts:meetings:20180126:se_classification
```
Comment (tunnell) [inline lax/lichens/sciencerun0.py]: Can you add just two sentences on how the cut works here?

File: lax/lichens/sciencerun0.py [hunk 2]
```diff
@@ -873,3 +875,35 @@ def _process(self, df):
...
+
+    def _process(self, df):
+
+        # Random forest classifier
+        forest_filename = os.path.join(DATA_DIR, 'XENON1T_random_forest_peak_classifier_01262018.pkl')
+        forest_load = pickle.load(open(forest_filename, 'rb'))  # noqa
+
+        # Gradient Boosted Decesion Tree classifier
+        gbdt_filename = os.path.join(DATA_DIR, 'XENON1T_gradient_bdt_peak_classifier_01262018.pkl')
+        gbdt_load = pickle.load(open(gbdt_filename, 'rb'))  # noqa
```
Comment (tunnell) [inline lax/lichens/sciencerun0.py:898]: Do you just get pickle security warnings?

**Review by tunnell — APPROVED** [review]

**Comment by tunnell** [comment]
You can stop those security warnings with # pylint disable:

https://stackoverflow.com/questions/4341746/how-do-i-disable-a-pylint-warning

**Comment by pdeperio** [comment]
@tunnell how do we apply that fix to Codacy here?

File: lax/lichens/sciencerun0.py (hunk 2 above)
Comment (feigaodm) [reply lax/lichens/sciencerun0.py:898]: Yeah, only saw those pickle warning. There is sth called __class__ not ... We discussed this in the [Katrina's SE PR too](https://github.com/XENON1T/hax/pull/203).

**Comment by feigaodm** [comment]
Does it make sense to merge it in for ease of use? We can update it later if it will change with more and cleaner training samples.

**Comment by pdeperio** [comment]
@tunnell oh I see what you did [here](https://github.com/XENON1T/hax/pull/203/files#diff-2bd0b79edcbeb4da3eefb71a9aaa0476R103). But we can't have this next to every single class definition now. Still need to figure out what has changed in the Codacy check...

**Pull request merged**